README.md

```diff
@@ -173,6 +180,62 @@ Kind: *Trusted Indication*.
 
 Errors:
 
+### `Staking`
+
+Send staking related messages. All the messages should be executed sequentially and applied atomically.
+
+- `messages: Vec<StakingMssage>`
+
+Kind: *Meta*.
+
+Errors:
+
+#### `StakingMessage` Types
+
+##### `Bond`
+
+- `value: Compact`: The amount of token to bond.
+
+Kind: *Instruction*.
+
+##### `Unbond`
+
+- `value: Compact`: The amount of token to unbond.
+
+Kind: *Instruction*.
+
+##### `Rebond`
```

Comment:
FYI if the goal is to be general: this is assuming the existence of a bonding duration in which you can rebond your tokens. While unlikely, some chains might not have this. Or, if this applies also to the notion of _delegators_, they usually don't have any bonding duration applied.

Reply:
XCM aims to be general, exhaustive and composable.